uiplib/Wallpaper.py

```diff
@@ -52,10 +52,10 @@ def set_wallpaper_osx(self, filename):  # pragma: no cover
     def get(self):
         """Get file address of current Wallpaper."""
         if sys.platform.startswith('win32'):   # pragma: no cover
-            return self.get_wallpaper_windows()
+            return [self.get_wallpaper_windows()]
 
         elif sys.platform.startswith('linux'):
-            return self.get_wallpaper_linux()
+            return [self.get_wallpaper_linux()]
```

Comment:
```python
return gsettings.get_string('picture-uri')[7:]
```
means it is already returning a list. Right?

Reply:
no a string

Reply:
see tests

Reply:
Oh. Okay!